pkg/bff/collaborators.go

```diff
@@ -78,6 +79,40 @@ func (s *Server) AddCollaborator(c *gin.Context) {
 	c.JSON(http.StatusOK, collaborator)
 }
 
+// ListCollaborators lists all the collaborators on the user's organization. The user
+// must have the read:collaborators permission to make this request.
+func (s *Server) ListCollaborators(c *gin.Context) {
+	var (
+		err          error
+		org 		*models.Organization
+	)
+
+	// Fetch the organization from the claims
+	// NOTE: This method handles the error logging and response
+	if org, err = s.OrganizationFromClaims(c); err != nil {
+		return
+	}
+
+	// Build the response from the internal map
+	out := &api.ListCollaboratorsReply{
+		Collaborators: make([]*models.Collaborator, 0),
+	}
+
+	for _, collab := range org.Collaborators {
+		if err = s.LoadCollaboratorDetails(collab); err != nil {
+			log.Error().Err(err).Str("collabID", collab.Key()).Msg("could not load collaborator details from Auth0")
```

Comment:
Should we return an error here? I'm leaning towards no, because at the worst case we just have stale data.

Reply:
If there are no collaborators on the organization, then the organization is marked for deletion; if the collaborators are being listed, it implies there is at least one -- the user who is listing the collaborators, right?